decidim-proposals/app/views/decidim/proposals/admin/proposals/show.html.erb

```diff
@@ -107,7 +107,7 @@
             <% proposal_meetings.each do |meeting| %>
               <% presented_meeting = present(meeting) %>
               <li>
-                <%= link_to presented_meeting.title, presented_meeting.profile_path %>
+                <%= link_to decidim_escape_translated(meeting.title).html_safe, presented_meeting.profile_path %>
```

Comment:
It seems that link_to and decidim_escape_translated do not play very well.  I would leave it with html_safe to avoid double escaping. 
![image](https://github.com/decidim/decidim/assets/105683/d05ac82a-8b89-4e76-8d87-373d83701a23)

Reply:
Ok, you won this one :trollface:

Reply:
I am assuming that this remains as i have proposed. Reopen if you have other opinion.